docs-markdown/src/helper/common.ts

```diff
@@ -186,7 +186,6 @@ export function hasValidWorkSpaceRootPath(senderName: string) {
 
 export async function insertContentToEditor(
 	editor: vscode.TextEditor,
-	senderName: string,
```

Comment:
I removed this parameter which caused such a large diff. But the change was very small because this parameter is unused.

Reply:
Nice clean up!